### ISSUE TITLE
Collect Card Payment w/ simulated device triggers PaymentIntent error

### ISSUE THREAD

**Issue by trag-stripe** [stripe/stripe-terminal-js-demo#53]
Having an issue running the sample - hoping it's something basic I'm missing.

**Steps to reproduce:**

1. Run `example-terminal-backend` on local machine
2. Run `example-terminal-js-client` locally & enter the localhost URL from the backend
3. Discover devices and selected the "simulated device" option
4. Under the _Common Workflows_ section, select collect card payment
5. Response is the following error:

```
EXCEPTION
Request Failed: Error creating PaymentIntent! Received unknown parameter: allowed_source_types
```

Thanks!

@trag

**Comment by esandler-stripe** [comment]
@trag-stripe Ah I think this is a bug in the example-terminal-backend where we need to set the Stripe Version header in order to make sure this works correctly. Thanks for flagging

**Comment by trag-stripe** [comment]
Ok cool - let me know what/when to do and I'll test again.
Thanks!

**Comment by danj-stripe** [comment]
Fixed by stripe/example-terminal-backend#8, which sets the stripe version explicitly.

Thanks for reporting!
